src/main/java/bot/Sailfish.java

```diff
@@ -0,0 +1,198 @@
+package bot;
```

Comment:
This class looks a bit messy. Maybe you can separate it to several classes (controller + task manager)